backend/cmd/headlamp.go

```diff
@@ -120,6 +120,10 @@ type OauthConfig struct {
 	Ctx      context.Context
 }
 
+type statelessClientConfig struct {
```

Comment:
I think we can reuse the `clientConfig` struct here instead of this new one as it is the same.